qa/workunits/rbd/rbd-nbd.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -ex
 
-. $(dirname $0)/../ceph-helpers.sh
+source $CEPH_ROOT/qa/standalone/ceph-helpers.sh
```

Comment:
I'd prefer to have these all remain relative paths -- ```CEPH_ROOT``` has a different connotation to me than the root directory of the git repo. See this same script for an example of how it uses ```CEPH_ROOT``` to think  it's the build directory.